### DISCUSSION TITLE
14. March 2023

### DISCUSSION THREAD

**Discussion by dimakuv** [gramineproject/gramine#1219] [Meeting notes]
### Agenda

*(please write your proposed agenda items in comments under this discussion)*

- [ ] gRPC integration with Gramine - Leila
- [x] Discuss moving "Users of Gramine" out of technical docs

(This was a quick meeting, since the first topic was not discussed.)

#### Miscellaneous

- Mona: We should move "Users of Gramine" out of technical docs.
   - Michal, Woju: yes, we should.
   - When we move "Users" to the web site, how do we want it to look? Do we just keep the names & logos, or do they provide the text?
      - (no conclusion on this)

- What is the status of the Gramine web site?
   - Mona: I'm not sure, probably the ball is on our side.

**Comment by monavij** [comment]
gRPC integration with Gramine - Leila
Discuss moving "Users of Gramine" out of technical docs
